docs/get-started/how-it-works.md

```diff
@@ -5,12 +5,8 @@ This involves three steps:
 
 ## Step 1: Build an embedding model
 
-Finetuner interprets the architecture of an existing (pre-trained model) which we call backbone.
-Those model might not be an embedding model upfront and the architecture not suitable for training it to encode data into embeddings.
-Therefore, Finetuner removes the default *head*, applies *pooling* and freezes layers that do not need to be trained.
-
 Finetuner takes an existing, pre-trained model, typically called the __backbone__, and analyzes its architecture.
-If this model does not already produce embeddings, Finetuner is able to remove the default *head* (the last layers of the network), add new projection layers, apply *pooling*, and freeze layers that do not need to be trained.
+If this model does not already produce embeddings or the architecture is not suitable for training it, Finetuner is able to remove the default *head* (the last layers of the network), add new projection layers, apply *pooling*, and freeze layers that do not need to be trained.
```

Comment:
```suggestion
If this model does not already produce embeddings or the architecture is not suitable for training, Finetuner is able to remove the default *head* (the last layers of the network), add new projection layers, apply *pooling*, and freeze layers that do not need to be trained.
```

Reply:
The model might already produce embeddings, but the architecture might not be suitable for the desired training.